tests/integration/MediaWiki/Api/EditEndpointTest.php

```diff
@@ -25,34 +28,52 @@ class EditEndpointTest extends \MediaWikiIntegrationTestCase {
 
 	use HandlerTestTrait;
 
-	private const URL_PROPERTY = 'P1';
+	/** @var PropertyId URL_PROPERTY */
+	private $URL_PROPERTY = 'P1';
+
+	/** @var PropertyId URL_PROPERTY */
+	private $ITEM_PROPERTY = 'P2';
+
 	private const MISSING_PROPERTY = 'P1000';
 
 	protected function setUp(): void {
 		parent::setUp();
 		$this->tablesUsed[] = 'page';
+		$this->tablesUsed[] = 'externallinks';
 		$this->tablesUsed[] = 'wb_property_info';
-	}
+		$this->tablesUsed[] = 'wb_id_counters';
 
-	private function newHandler() {
-		$propertyDataTypeLookup = new InMemoryDataTypeLookup();
+		/** @var PropertyId URL_PROPERTY */
+		WikibaseRepo::getDefaultInstance()->getEntityStore()->saveEntity(
+			new Property( new PropertyId( $this->URL_PROPERTY ), null, 'url' ),
+			__METHOD__,
+			User::newSystemUser( 'TestUSer' ),
```

Comment:
Uppercase S in TestUSer looks accidental?

Also, this should probably use `MediaWikiIntegrationTestCase::getTestUser()->getUser()` anyways.

Reply:
thanks